### DISCUSSION TITLE
How can I determine the source of local weather data?

### DISCUSSION THREAD

**Discussion by csmarshall** [Pirate-Weather/pirateweather#188] [Q&A] [answered]
Generally the forecasts and current data are correct, however currently there seems to be a roughly 15 degree F difference between what Pirate Weather reports for my location and openweather

Pirate weather:
<img width="355" alt="Screenshot 2024-04-15 at 13 13 04" src="https://github.com/Pirate-Weather/pirateweather/assets/1200112/cc928711-a273-4d74-accb-ad646fb8b8ab">

OpenWeather:
<img width="374" alt="Screenshot 2024-04-15 at 13 13 33" src="https://github.com/Pirate-Weather/pirateweather/assets/1200112/93ad54ec-4e9e-4c9b-b678-af601f83ef7d">

Apple Weather:
<img width="188" alt="Screenshot 2024-04-15 at 13 18 20" src="https://github.com/Pirate-Weather/pirateweather/assets/1200112/53d8e2f6-cc58-494f-8251-fc0f8b3d8c23">

I'm assuming my local data source for pirate weather isn't updating or reporting old data, but I am unsure how to find the right source? 

Pretty savvy so I could dig into the exact values etc, but thought a surface understanding would help me dig further. 

Thanks,

**Comment by cloneofghosts** [comment, marked as answer]
The weather models used to provide the forecast for your location can be found under the sources section in the flags block if you're querying the API directly. If you're using MerrySky the sources are listed directly after the data attribution under the forecasts.

For your location the HRRR weather model is used for your location for the current conditions and the forecasts. Currently the weather API doesn't incorporate any station data into its current conditions at the moment.

**Reply by cloneofghosts** [reply]
I should also note that NOAA was having outages two days ago and yesterday which may have impacted the accuracy of HRRR and since everything is back online it should be better now.

**Reply by alexander0042** [reply]
Yes, definitely this, this extended NOAA outage really whacked things around the last few days. Otherwise, the docs have details on where each parameter is coming from, but in general it's HRRR/NBM for North America, and GFS/GEFS outside of that
